docker/lr-metrics/Dockerfile

```diff
@@ -16,9 +16,10 @@ ENV PATH=/miniconda/bin/:/miniconda/envs/lr-metrics/bin/:/root/google-cloud-sdk/
 
 # install conda packages
 COPY ./environment.yml /
+RUN conda install -n base conda-libmamba-solver && conda config --set solver libmamba
 RUN conda env create -f /environment.yml && conda clean -a
 
-# install gatk
+# install super-special version of gatk
```

Comment:
I've created a ticket for this to help us track #427

Reply:
Current version eliminates it.